Workbooks/OrcaAlerts.json

```diff
@@ -0,0 +1,113 @@
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "visualization": "linechart",
+        "chartSettings": {
+          "seriesLabelSettings": [
+            {
+              "seriesName": "3.0",
+              "label": "Hazardous"
+            },
+            {
+              "seriesName": "2.0",
+              "label": "Imminent Compromise"
+            },
+            {
+              "seriesName": "1.0",
+              "label": "Compromise"
+            }
+          ]
+        }
+      },
+      "customWidth": "50",
+      "name": "Alerts over time"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "\n\nOrcaAlerts_CL \n| summarize count(alert_id_s) by time_t , cloud_provider_id_s \n\n\n\n",
+        "size": 0,
+        "timeContext": {
+          "durationMs": 86400000
+        },
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "visualization": "linechart"
+      },
+      "customWidth": "50",
+      "name": "query - 4"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "OrcaAlerts_CL \n| summarize count(alert_id_s) by asset_type_s \n",
+        "size": 3,
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "visualization": "piechart"
+      },
+      "customWidth": "33",
+      "name": "query - 2"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "OrcaAlerts_CL \n| summarize count(alert_id_s) by alert_type_s ",
+        "size": 3,
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "visualization": "piechart",
+        "tileSettings": {
+          "showBorder": false
+        },
+        "graphSettings": {
+          "type": 0
+        },
+        "mapSettings": {
+          "locInfo": "LatLong",
+          "sizeSettings": "score_d",
+          "sizeAggregation": "Sum",
+          "legendMetric": "score_d",
+          "legendAggregation": "Sum",
+          "itemColorSettings": {
+            "type": "heatmap",
+            "colorAggregation": "Sum",
+            "nodeColorField": "score_d",
+            "heatmapPalette": "greenRed"
+          }
+        }
+      },
+      "customWidth": "33",
+      "name": "query - 1"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "OrcaAlerts_CL \n| summarize count() by tostring(score_d)\n",
+        "size": 3,
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "visualization": "piechart"
+      },
+      "customWidth": "33",
+      "name": "query - 3"
+    }
+  ],
+  "fallbackResourceIds": [
+    "/subscriptions/0fc23999-0bae-4c46-8483-55f743b8f301/resourceGroups/Sentinel_RG/providers/Microsoft.OperationalInsights/workspaces/SentielInt"
```

Comment:
This should not be here as it is a specific resource.
I guess it is your workspace.
This workbook will be a template for the community and customers of course will not have permissions to this workspace.